JSDemos/Demos/List/ListEditingAndAPI/React/App.js

```diff
@@ -10,63 +10,46 @@ class App extends React.Component {
   constructor() {
     super();
     this.state = {
-      allowDeleting: false,
+      allowDeletion: false,
       deleteType: 'toggle',
```

Comment:
shouldn't we replace "deleting type" with "itemDeleteMode" everywhere?

Reply:
Yes. But somebody already did it for me.
All labels above SelectBoxes are "Item delete mode". I though this task is about labels (what customers see on a page)

Reply:
Customers see the code too)